core/src/replay_stage.rs

```diff
@@ -2896,6 +2897,34 @@ impl ReplayStage {
                     (bank.slot(), bank.hash()),
                     Some((bank.parent_slot(), bank.parent_hash())),
                 );
+                // If the block does not have at least 64 shreds in the last FEC set, mark
+                // it as duplicate, effectively removing it from fork choice.
```

Comment:
```suggestion
                // it as invalid, effectively removing it from fork choice.
```

right? could be for reasons other than fuckery